pipelines/clone/03-clone_server.yaml

```diff
@@ -0,0 +1,43 @@
+---
+- name: install server
+  hosts:
+   - "{{ forklift_server_name }}"
+  become: yes
+  vars_files:
+    - ../vars/clone_base.yml
+    - ../vars/repos_staging.yml
+  tasks:
+    - name: create backup dir
+      file:
+        path: "{{ backup_dir }}"
+        state: directory
+    - name: download backup files
+      get_url:
+        dest: "{{ backup_dir }}/{{ item }}"
+        url: "{{ clone_source_url }}/{{ pipeline_version }}-{{ clone_type }}/{{ item }}"
+      loop: "{{ backup_files[clone_type] }}"
+    - name: install satellite-clone
+      package:
+        name:
+          - ansible
+          - git
+          - rubygem-foreman_maintain
+        state: present
+    - name: clone satellite-clone
+      git:
+        repo: https://github.com/evgeni/satellite-clone
+        dest: /root/satellite-clone
+        version: mock
```

Comment:
that's a slightly modified version of `sat-clone` which is capable to detect a Katello nightly as "Satellite 6.10"

Reply:
NOT NEEDED ANYMORE!